### DISCUSSION TITLE
Guess country

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#596] [Galaxy Trivia] [answered]
🔷⬇️⬜🟥
⬛🟨❌🟩
🟥☀️

**Comment by peterbaccaro** [comment]
1) CZECH REPUBLIC
2) JAMAICA
3) MACEDONIA

![flags_2](https://user-images.githubusercontent.com/74666079/233477657-ad97020a-c882-4ffd-bd84-6f64a074fd13.jpg)

**Comment by thuongtruong1009** [comment, marked as answer]
I believe the country you are referring to is Belgium.
